bundle/edu.gemini.pot/src/main/java/edu/gemini/spModel/gemini/altair/AltairParams.java

```diff
@@ -44,14 +44,7 @@ public String displayValue() {
         }
 
         public String sequenceValue() {
-            return _displayValue;
-        }
-
-        /**
-         * Return a Wavelength by index *
-         */
-        static public Wavelength getWavelengthByIndex(int index) {
-            return SpTypeUtil.valueOf(Wavelength.class, index, DEFAULT);
+            return name();
```

Comment:
Makes sense, you are right. I'll create a separate string field for the sequence value that you will need to provide at construction.